tests/test_all_fields.py

```diff
@@ -167,10 +161,7 @@
         'plan',
     },
     'invoices': {
```

Comment:
Explain this code removal

Reply:
Added the missing subfields for the `discount `schema. The `finalized_at` field is deprecated in the new API version 2020-08-27, hence added in the `FIELDS_TO_NOT_CHECK ` map. Moreover, for the `created`, converted the expected value to timestamp during the assertion, thus `created` will no longer fail. Hence removed the following fields from this map.